eng/Versions.props

```diff
@@ -10,7 +10,7 @@
     <VersionSDKMinor>1</VersionSDKMinor>
     <VersionFeature>00</VersionFeature>
     <VersionPrefix>$(VersionMajor).$(VersionMinor).$(VersionSDKMinor)$(VersionFeature)</VersionPrefix>
-    <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel>rc</PreReleaseVersionLabel>
     <PreReleaseVersionIteration>1</PreReleaseVersionIteration>
     <PreReleaseBrandingLabel>rc</PreReleaseBrandingLabel>
```

Comment:
You can get rid of this too

Reply:
So is `PreReleaseBrandingLabel` not used at all?

Reply:
Correct, it was just added to this repo in the previous branding PR.